### ISSUE TITLE
Broken CodeSandbox link

### ISSUE THREAD

**Issue by nbdn** [uber-web/baseui#1547]
## Current Behavior
On the official documentation (https://baseweb.design/components/button-group/), when clicking an "Edit on CodeSandbox", the link does nothing. I have the following error in my console : 

> baseweb.design/:1 Access to fetch at 'https://codesandbox.io/api/v1/sandboxes/define?json=1' from origin 'https://baseweb.design' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

## Expected Behavior
The link must open a new tab on CodeSandbox.

## Your Environment

MacOS Mojave 10.14.5
Chrome Version 75.0.3770.100 (64 bits)

- [x] I have searched the [issues](https://github.com/uber-web/baseui/issues) of this repository and believe that this is not a duplicate.

**Comment by gergelyke** [comment]
Hi @nbdn - I could not reproduce this. Any chance you can try in an incognito window?

**Comment by nbdn** [comment]
Hi @gergelyke,

It was network related.

I was on a public network, I changed it to my 4G phone network and it works now. I think the issue should be "wontfix".

Thanks for your interest

**Comment by gergelyke** [comment]
Happy to hear that it works! Please let us know if you run into further questions / problems!
